src/globus_sdk/transport/encoders.py

```diff
@@ -1,4 +1,7 @@
+from typing import Any, Dict, Optional
+
 import requests
+from requests.models import Request
```

Comment:
Was it necessary to switch from `requests.Request` to `requests.models.Request`?
We should prefer the top-level namespace of external packages since there's a stronger expectation of stability there (even across major versions).